mk/spksrc.common.mk

```diff
@@ -14,7 +14,7 @@ MSG = echo "===> "
 RUN = cd $(WORK_DIR)/$(PKG_DIR) && env $(ENV)
 
 # Pip command
-PIP ?= pip
+PIP ?= pip #define PIP in your makefile if you need pip3
 PIP_WHEEL = $(PIP) wheel --no-use-wheel --no-deps --wheel-dir $(STAGING_DIR)/wheelhouse
```

Comment:
Yes, I agree. The documentation is never going to cover everything, as in: after you read it, you can make a package. It won't be an instruction on how to cross-compile an arbitrary piece of software. That's simply not feasible; I would end up writing a book on cross-compiling: no thanks ;) 
Realistically speaking, the starting point will be to explain the entries in a Makefile, such as content, format, purpose, and that's mostly spksrc specific stuff. We'll see where we go from there.

Back to the subject at hand: I'll probably add it to the Python/wheel wiki article, that's a sensible place to put it (https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-solutions). I was working on an update of the doc, so I'll fit it in when I get to finishing it.